src/middlewared/middlewared/etc_files/local/smb4.conf

```diff
@@ -362,7 +362,7 @@
             pc = {}
             if db['truenas_conf']['smb_ha_mode'] == 'UNIFIED' and db['truenas_conf']['failover_status'] != 'MASTER':
                 stub_config = {
-                    'netbios name': f"{db['gc']['hostname_virtual']}_PASSIVE",
+                    'netbios name': f"{db['gc']['hostname_virtual']}_STANDBY",
```

Comment:
@anodos325 can we do this safely?

Reply:
@themylogin yes, you can do this safely without effecting anything.